pages/treeherder.py

```diff
@@ -152,6 +207,22 @@ def open_perfherder_page(self):
     def open_repos_menu(self):
         self.find_element(*self._repos_menu_locator).click()
 
+    def pin_job_and_enter_bug_number(self, bug_number):
+        el = self.find_element(*self._result_sets_locator)
+        el.send_keys('b')
+        el = self.find_element(*self._related_bug_input_locator)
+        el.send_keys(bug_number + Keys.RETURN)
+
+    def pin_job_and_enter_classification(self, classification):
+        el = self.find_element(*self._result_sets_locator)
+        el.send_keys('c')
+        el = self.find_element(*self._classification_comment_locator)
+        el.send_keys(classification + Keys.RETURN)
+
+    def pin_random_job(self, driver):
+        random_job = random.choice(self.all_jobs).click()
+        ActionChains(driver).key_down(Keys.CONTROL).click(random_job).perform()
```

Comment:
I'm not sure about moving this, because there is a problem with `driver` and with ActionChains.
Would it be better to leave it here? As `pin_using_spacebar`?